Dockerfile.rhel8

```diff
@@ -7,7 +7,7 @@ FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.9 AS tuned
 WORKDIR /root
 COPY assets /root/assets
 RUN INSTALL_PKGS=" \
-      gcc git rpm-build make desktop-file-utils patch dnf-plugins-core \
+      gcc git rpm-build make desktop-file-utils patch dnf-plugins-core kernel-tools-libs-4.18.0-305.3.1.el8_4.x86_64 kernel-tools-4.18.0-305.3.1.el8_4.x86_64 python3-perf-4.18.0-305.3.1.el8_4.x86_64 \
```

Comment:
`FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.9 AS tuned` tuned here is just a "build" container for building TuneD rpms.  I do not believe this has any influence over what's shipped in the resulting operator/operand image.